src/Symfony/Component/Form/FormBuilder.php

```diff
@@ -73,6 +73,8 @@ public function add($child, $type = null, array $options = [])
             throw new UnexpectedTypeException($child, 'string or Symfony\Component\Form\FormBuilderInterface');
         }
 
+        $child = (string) $child;
```

Comment:
what's the value of this?

Reply:
Since we now have strings everywhere, I decided to typecast explicitly here because we won't be able to add a type declaration on this method in `5.0` (`$child` might be an instance of `FormInterface` as well).
Although now I got that arrays support `'0'` and `0` in the same way, right? So it does not matter here.